### PR TITLE
Github conventions

### PR DESCRIPTION
**Pull request by stian-sandvold**
Based on existing unwritten guidelines, and discussion from the backend meeting.

Any errors, wrong statements or suggestions on improvements are welcome

### CODE REVIEW

File: guides/github.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+1. The author is primarily responsible for merging their own PRs
+2. Merging should only happen when all comments have been resolved
+3. Auto-merging PRs should be used when the author is confident no further work is required
+4. Other developers may only merge PRs they have not authored when all checks are OK, two approvals exist and no comment lies unresolved.
+
+
+**GitHub conventions**
+1. PR titles must follow the naming convention, prepending the title with "fix|feat|chore|test|ci|", based on the content of their PR.
+2. All PRs (omitting Draft PRs) must have a description explaining the content of the PR.
+3. All branches should be named on the format: "<Jira-issue-number>[_<version>]". Example DHIS2-123 and DHIS2-123-2.40.
```
Comment (jbee) [inline guides/github.md:32]: format and example disagree on the `_`/`-`

File: guides/github.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+4. PRs should be assigned to at least 2 reviewers, or a team of reviewers
```
Comment (jbee) [inline guides/github.md:33]: So far in platform we use no explicit assignment for "anyone in the team". This change is fine but we then need to make sure a team exist so we can continue with a form of non-specific assignments.

**Review by jbee — APPROVED** [review]

**Review by netroms — APPROVED** [review]

File: guides/github.md (hunk 1 above)
Comment (stian-sandvold) [inline guides/github.md:32]:
```suggestion
3. All branches should be named on the format: "<Jira-issue-number>[_<version>]". Example DHIS2-123 and DHIS2-123_2.40.
```

File: guides/github.md (hunk 2 above)
Comment (stian-sandvold) [reply guides/github.md:33]: I got that on my list, will set up teams on GitHub soon :)

File: guides/github.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+1. The author is primarily responsible for merging their own PRs
+2. Merging should only happen when all comments have been resolved
+3. Auto-merging PRs should be used when the author is confident no further work is required
+4. Other developers may only merge PRs they have not authored when all checks are OK, two approvals exist and no comment lies unresolved.
+
+
+**GitHub conventions**
+1. PR titles must follow the naming convention, prepending the title with "fix|feat|chore|test|ci|", based on the content of their PR.
+2. All PRs (omitting Draft PRs) must have a description explaining the content of the PR.
+3. All branches should be named on the format: "<Jira-issue-number>[_<version>]". Example DHIS2-123 and DHIS2-123_2.40.
```
Comment (jbee) [inline guides/github.md:32]: I found a small issue with the pattern using version as suffix. 
If you have multiple PRs for same issue number you have to add another suffix, like `DHIS2-1234-additional`. If you then backport this if is unclear what to do. `DHIS2-1234-2.39-additional`?
Therefore I suggest making the version a prefix: `2.39_DHIS2-1234-additional`. That way it is always `<version>_<unversioned-branch-name>`.

**Pull request merged**